src/protocols/interaction_model/StatusCodeList.h

```diff
@@ -65,4 +65,6 @@ CHIP_IM_STATUS_CODE(NoUpstreamSubscription, NO_UPSTREAM_SUBSCRIPTION, 0xc5)
 CHIP_IM_STATUS_CODE(NeedsTimedInteraction , NEEDS_TIMED_INTERACTION , 0xc6)
 CHIP_IM_STATUS_CODE(UnsupportedEvent      , UNSUPPORTED_EVENT       , 0xc7)
 CHIP_IM_STATUS_CODE(PathsExhausted        , PATHS_EXHAUSTED         , 0xc8)
+CHIP_IM_STATUS_CODE(WriteIgnored          , WRITE_IGNORED           , 0xc9)
```

Comment:
This is fine, but maybe add a comment that this is a non-spec error code we use internally and if 0xC9 ever gets assigned to a spec error code this value can be changed to something else?